engine/schema/src/main/resources/META-INF/db/schema-410to420.sql

```diff
@@ -2065,7 +2065,7 @@ alter table `cloud`.`vpc_gateways` add column network_acl_id bigint unsigned def
 update `cloud`.`vpc_gateways` set network_acl_id = 2;
 
 
-INSERT IGNORE INTO `cloud`.`configuration` VALUES ('Advanced', 'DEFAULT', 'VpcManager', 'blacklisted.routes', NULL, 'Routes that are blacklisted, can not be used for Static Routes creation for the VPC Private Gateway');
+INSERT IGNORE INTO `cloud`.`configuration` VALUES ('Advanced', 'DEFAULT', 'VpcManager', 'blacklisted.routes', NULL, 'Routes that are denied, can not be used for Static Routes creation for the VPC Private Gateway');
```

Comment:
this should be done in a 4.14 to 4.15 upgrade file and not in this old one.

Reply:
4.16 now. Also, we are not changing name for this config and just the description? `blacklisted.routes`